src/matrand_model.jl

```diff
@@ -0,0 +1,82 @@
+export mat_rand_model
```

Comment:
There is no mat_rand_model maybe you mean:
```suggestion
export matrix_completion_model
```